test/cloudant.test.js

```diff
@@ -70,41 +70,6 @@ describe('cloudant connector', function() {
     });
   });
 
-  describe('updateAll and updateAttributes', function() {
```

Comment:
If the `updateAll and updateAttributes` test was a duplicate of the `updateAll` tests, then that definitely needs fixing, but I would rewrite it to be an isolated test of `updateAttributes` exclusively, rather than deleting it.